src/plugins/messenger/MessengerPreview/components/MessengerImage/components/LightBoxHeader.tsx

```diff
@@ -0,0 +1,107 @@
+import { useRef } from "react";
+import { useMessangerImageContext } from '../MessangerImageContext';
+import { MessagePluginFactoryProps } from "../../../../../../common/interfaces/message-plugin";
+import CloseIcon from '../../../../../../webchat-ui/assets/baseline-close-24px.svg';
+import DownloadIcon from '../../../../../../webchat-ui/assets/baseline-download-24px.svg';
+
+export const getLightBoxHeader = ({ React, styled }: MessagePluginFactoryProps) => {
+
+    const Header = styled.div(({ theme }) => ({
+        position: 'absolute',
+        top: 0,
+        height: '56px',
+        width: '100%',
+        backgroundColor: theme.primaryColor,
+        overflow: 'hidden',
+        display: 'flex',
+        justifyContent: 'space-between'
+    }))
+
+    const Caption = styled.div({
+        display: 'flex',
+        alignItems: 'center',
+        marginLeft: '15px',
+        color: 'white',
```

Comment:
We should use primaryContrastColor from the theme, as the text will be hard to read on light backgrounds (White or other light colors might also be the primary color).